doc/README.md

```diff
@@ -0,0 +1,97 @@
+# Cothority
+
+The Cothority (Collective Authority) is a project maintained by the
+[DEDIS](https://dedis.epfl.ch) lab. Its goal is to collect projects that
+handle aspects of collective authorities to support scalable, self-organizing
+communities. This document collects web-pages, blog-posts, wikis,
+godocs, papers and more in a central place as a reference.
+
+## Program Code
+
+At DEDIS we like to take code from research projects and turn it into usable
+code for other projects, too. We program most of the time using
+[golang][https://golang.org] because it is a system language that is easy to
+learn but fast. Our main efforts are on the kyber library and the cothority.
```

Comment:
If we are changing the title then this paragraph may need to be changed too to make it flow a bit better. Perhaps to something like: "Software quality is often emphasised in our research. Therefore, the software artifact is often useful for the industry, the general public and future research projects. We organise it into three key repositories below."